### ISSUE TITLE
Feature request: Dockerfile

### ISSUE THREAD

**Issue by reedog117** [Jalle19/node-ffmpeg-mpegts-proxy#41]
Could you create a Dockerfile to launch a lightweight instance of this? (kinda similar to the arut/nginx-rtmp docker image in Docker Hub)

This would make things very easy for a one-step installation.

**Comment by Jalle19** [comment]
@reedog117 can you try https://github.com/Jalle19/node-ffmpeg-mpegts-proxy/pull/45?

**Comment by accessone67** [comment]
Just what I looked for. Thank you works great for me :)

**Comment by Jalle19** [comment]
Cool. I'll push it to the Docker Hub at some point, I'll ping you here when it's done.

**Comment by reedog117** [comment]
Just tried it and it works for me as well. One thing I did notice, though.. if I change the sources.json outside the container, the app doesn't notice the change and I have to restart the container. Not sure if this is a Docker limitation, though.

**Comment by Jalle19** [comment]
Yeah I noticed the same thing, except it usually worked *once*, but not again after that. I'm not familiar enough with Docker to know whether it's a Docker thing or not unfortunately.

**Comment by Jalle19** [comment]
Then again, if you have something in place that programmatically updates sources.json it probably isn't a too big of a deal to add a "docker restart" to it.

**Comment by Jalle19** [comment]
I published the Docker image to Docker Hub: https://hub.docker.com/r/jalle19/node-ffmpeg-mpegts-proxy/. The README is updated to reflect that you don't have to build the container yourself anymore.
